src/components/Typography/Typography.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import TypographyMaterial, { TypographyProps as TypographyPropsMaterial } from '@mui/material/Typography';
+import classNames from 'classnames/bind';
+
+import styles from './Typography.module.css';
+
+const cn = classNames.bind(styles);
+
+export type TypographyProps = {
+  className?: string;
+} & Pick<TypographyPropsMaterial, 'children' | 'className' | 'variant' | 'align'>;
```

Comment:
you can delete className here because you already declareted it at the top

Reply:
Done